test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/RiveAnimationTest.cs

```diff
@@ -0,0 +1,163 @@
+                Text = "Bounce"
+            };
+            bounceButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetAnimation("bouncing", true);
+            };
+
+            brokeButton = new Button()
+            {
+                Size = new Size(200, 100),
+                Position = new Position(200, 100),
+                Text = "Broken"
+            };
+            brokeButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetAnimation("broken", true);
+            };
+
+            fillButton = new Button()
+            {
+                Size = new Size(200, 100),
+                Position = new Position(0, 200),
+                Text = "Fill"
+            };
+            fillButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetFillColor("grillFillColor", new Color(1.0f, 0.0f, 0.0f, 1.0f));
+            };
+
+            strokeButton = new Button()
+            {
+                Size = new Size(200, 100),
+                Position = new Position(200, 200),
+                Text = "Stroke"
+            };
+            strokeButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetStrokeColor("grillStrokeColor", new Color(0.0f, 255.0f, 0.0f, 255.0f));
+            };
+
+            opacityButton = new Button()
+            {
+                Size = new Size(200, 100),
+                Position = new Position(400, 200),
+                Text = "Opacity"
+            };
+            opacityButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetOpacity("front_light", 0.3f);
+            };
+
+            scaleButton = new Button()
+            {
+                Size = new Size(200, 100),
+                Position = new Position(0, 300),
+                Text = "Scale"
+            };
+            scaleButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetScale("front_light", new Vector2(2.0f, 2.0f));
+            };
+
+            rotationButton = new Button()
+            {
+                Size = new Size(200, 100),
+                Position = new Position(200, 300),
+                Text = "Rotation"
+            };
+            rotationButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetRotation("front_light", new Degree(45.0f));
+            };
+
+            positionButton = new Button()
+            {
+                Size = new Size(200, 100),
+                Position = new Position(400, 300),
+                Text = "Position"
+            };
+            positionButton.Clicked += (object source, ClickedEventArgs args) =>
+            {
+                rav.SetPosition("front_light", new Vector2(100.0f, -50.0f));
+            };
+
+            NUIApplication.GetDefaultWindow().GetDefaultLayer().Add(rav);
```

Comment:
how about this?

private Window window = NUIApplication.GetDefaultWindow();
private Layer defaultLayer = window.GetDefaultLayer();
...
defaultLayer.Add(rav);
defaultLayer.Add(playButton);
...